tests/test_renderers/test_fixtures.py

```diff
@@ -86,11 +88,15 @@ def test_sphinx_directives(line, title, input, expected):
     # TODO test domain directives
     if title.startswith("SKIP"):
         pytest.skip(title)
+    if title.startswith("SPHINX3") and sphinx.version_info[0] < 3:
```

Comment:
your project doesn't truly depend on Sphinx until you have infrastructure with if/then statements that depend on the Sphinx version

Reply:
lol at least it doesn't affect any actual code though (just some unimportant things in the test)